src/config/types.ts

```diff
@@ -0,0 +1,43 @@
+import * as yup from "yup";
```

Comment:
just a nitpick, maybe we should call file `schemas.ts` cause the core here is our schema definitions?

Reply:
renamed